### PR TITLE
HW02 by Vladimir Sereda

### PR DESCRIPTION
**Pull request by VladimirSereda**

### CODE REVIEW

File: HW02/Vladimir_Sereda/vladimir_sereda_hw_02_t_01.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+logs = File.readlines('logs/logs_ts1.txt')
+
+def error_message(logs)
+  logs.select { |keyword| keyword.downcase =~ /error/ }
```
Comment (vladhilko) [inline HW02/Vladimir_Sereda/vladimir_sereda_hw_02_t_01.rb]: Should return only the first line with an error

Comment (VladimirSereda) [reply HW02/Vladimir_Sereda/vladimir_sereda_hw_02_t_01.rb]: Sorry my mistake, fixed

Comment (vladhilko) [reply HW02/Vladimir_Sereda/vladimir_sereda_hw_02_t_01.rb]: Try to update your file  `'logs/logs_ts1.txt'` with more data, you select more than one lines

Comment (VladimirSereda) [reply HW02/Vladimir_Sereda/vladimir_sereda_hw_02_t_01.rb]: Done

**Pull request merged**